src/main/java/flaxbeard/immersivepetroleum/common/blocks/metal/TileEntityPumpjack.java

```diff
@@ -22,574 +35,658 @@
+				getPos().getZ() >> 4);
 	}
-	
-	public int getResidualOil()
-	{
-		return PumpjackHandler.getResidualFluid(worldObj, getPos().getX() >> 4, getPos().getZ() >> 4);
+
+	public int getResidualOil() {
+		return PumpjackHandler.getResidualFluid(worldObj, getPos().getX() >> 4,
+				getPos().getZ() >> 4);
 	}
-	
-	public void extractOil(int amount)
-	{
-		PumpjackHandler.depleteFluid(worldObj, getPos().getX() >> 4, getPos().getZ() >> 4, amount);
+
+	public void extractOil(int amount) {
+		PumpjackHandler.depleteFluid(worldObj, getPos().getX() >> 4,
+				getPos().getZ() >> 4, amount);
 	}
-	
+
 	@Override
-	public void readCustomNBT(NBTTagCompound nbt, boolean descPacket)
-	{
+	public void readCustomNBT(NBTTagCompound nbt, boolean descPacket) {
 		super.readCustomNBT(nbt, descPacket);
 		boolean lastActive = wasActive;
 		this.wasActive = nbt.getBoolean("wasActive");
-		if (!wasActive && lastActive)
-		{
+		if (!wasActive && lastActive) {
 			this.activeTicks++;
 		}
 		state = null;
-		if (nbt.hasKey("comp"))
-		{
-			ItemStack stack = ItemStack.loadItemStackFromNBT(nbt.getCompoundTag("comp"));
-			
-			if (stack != null)
-			{
+		if (nbt.hasKey("comp")) {
+			ItemStack stack = ItemStack
+					.loadItemStackFromNBT(nbt.getCompoundTag("comp"));
+
+			if (stack != null) {
 				Block block = Block.getBlockFromItem(stack.getItem());
 				state = block.getDefaultState();
 			}
 		}
 	}
+
 	@Override
-	public void writeCustomNBT(NBTTagCompound nbt, boolean descPacket)
-	{
+	public void writeCustomNBT(NBTTagCompound nbt, boolean descPacket) {
 		super.writeCustomNBT(nbt, descPacket);
 		nbt.setBoolean("wasActive", wasActive);
-		
-		if (availableFluid() != null)
-		{
-			if (availableFluid().getBlock() != null)
-			{
+
+		if (availableFluid() != null) {
+			if (availableFluid().getBlock() != null) {
 				ItemStack stack = new ItemStack(availableFluid().getBlock());
 				NBTTagCompound comp = new NBTTagCompound();
 				stack.writeToNBT(comp);
 				nbt.setTag("comp", comp);
 			}
 		}
 	}
-	
+
 	@Override
-	public void update()
-	{
+	public void update() {
 		update(true);
 	}
 
-	public void update(boolean consumePower)
-	{
-		//System.out.println("TEST");
+	public void update(boolean consumePower) {
+		// System.out.println("TEST");
 		super.update();
-		if (worldObj.isRemote || isDummy())
-		{
-			if (worldObj.isRemote && !isDummy() && state != null && wasActive)
-			{
+		if (worldObj.isRemote || isDummy()) {
+			if (worldObj.isRemote && !isDummy() && state != null && wasActive) {
 				BlockPos particlePos = this.getPos().offset(facing, 4);
 				float r1 = (worldObj.rand.nextFloat() - .5F) * 2F;
 				float r2 = (worldObj.rand.nextFloat() - .5F) * 2F;
 
-				worldObj.spawnParticle(EnumParticleTypes.BLOCK_DUST, particlePos.getX() + 0.5F, particlePos.getY(), particlePos.getZ() + 0.5F, r1 * 0.04F, 0.25F, r2 * 0.025F, new int[] {Block.getStateId(state)});
+				worldObj.spawnParticle(EnumParticleTypes.BLOCK_DUST,
```

Comment:
Is there some way for me to get my hands on the original formatter file? This part was an accident. I'll reclone if needed, but that's a last resort.